### ISSUE TITLE
Search, filter, update buttons in library don't show after migrating searched title

### ISSUE THREAD

**Issue by rgryta** [tachiyomiorg/tachiyomi#6458]
### Steps to reproduce

1. Open library
2. Choose any title in the library
3. In the search bar start inputting the title
4. Open that title while in search mode
5. Migrate it (can be to the same source as it currently is)
6. Once migration is finished, click on back button
7. (optional) Empty the search bar - buttons are already missing, but there's a submenu to the top right now that seems to be working. However, it's still unexpected behavior cause it's not consistent (before opening the title there are normal separate buttons, not a submenu).
8. Close the search bar

### Expected behavior

Top bar should display Search, Filter/Sort/View, Update buttons again.

### Actual behavior

Top bar displays a three-dotted submenu that doesn't do anything when pressed.

### Tachiyomi version

0.12.3

### Android version

Android 11

### Device

Xiaomi Mi 9T Pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
